prow/plugins/label/label.go

```diff
@@ -96,20 +101,24 @@ func getLabelsFromREMatches(matches [][]string) (labels []string) {
 
 // getLabelsFromGenericMatches returns label matches with extra labels if those
 // have been configured in the plugin config.
-func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string) []string {
+func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string, invalidLabels *[]string) []string {
 	if len(additionalLabels) == 0 {
 		return nil
 	}
 	var labels []string
+	labelFilter := map[string]bool{}
+	for _, l := range additionalLabels {
+		labelFilter[strings.ToLower(l)] = true
```

Comment:
Nit: `k8s.io/apimachinery/pkg/util/sets.String` is a lot nicer to use and read :)